app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: float) -> "Distance":
```

Comment:
`other` argument could be `int` or `Distance` class instance as well. Besides that, write return type without the use of `""`. Change your type annotation here and in other methods as well to encounter that. In case you will face annotation errors, place `from __future__ import annotations` at the beginning of the file.